src/AutoScrum.AzureDevOps/Models/WorkItem.cs

```diff
@@ -15,6 +17,19 @@ public class WorkItem
         public DateTimeOffset? ChangedDate { get; set; }
         public DateTimeOffset? StateChangeDate { get; set; }
         public string Url { get; set; } = string.Empty;
+        public string? Blocked
+        {
+            get { return _blocked; }
+            set
+            {
+                _blocked = value;
+
+                // It turns out the value is either null/empty or "Yes"...
+                IsBlocked = !string.IsNullOrEmpty(_blocked);
+            }
+        }
+        public bool IsBlocked { get; set; }
```

Comment:
Should this not be `public bool IsBlocked => !string.IsNullOrEmpty(_blocked)`? Or is `IsBlocked` set on the frontend?

Reply:
This would also remove the code for the backing property